studio/.env

```diff
@@ -4,13 +4,13 @@ POSTGRES_PASSWORD=your-super-secret-and-long-postgres-password
 DEFAULT_ORGANIZATION_NAME=Default Organization
 DEFAULT_PROJECT_NAME=Default Project
 
-SUPABASE_URL=http://localhost:8000
-SUPABASE_PUBLIC_URL=http://localhost:8000
+NEXT_PUBLIC_SUPABASE_URL=http://localhost:8000
+NEXT_PUBLIC_SUPABASE_PUBLIC_URL=http://localhost:8000
```

Comment:
Can we keep both to stay on the safe side? I think `SUPABASE_URL` might still be used in some API routes. For example:

```bash
SUPABASE_URL=http://localhost:8000
SUPABASE_PUBLIC_URL=http://localhost:8000
NEXT_PUBLIC_SUPABASE_URL=$SUPABASE_URL
NEXT_PUBLIC_SUPABASE_PUBLIC_URL=$SUPABASE_PUBLIC_URL
```

Reply:
That's probably a good idea.

Reply:
I made the change

Reply:
Thanks! What are your thoughts on using the interpolated variables (`$SUPABASE_URL`) instead of duplicating the URLs to make updating them easier?

Reply:
Sorry, this is new to me so I missed that in your example. I updated it to use interpolation for the `NEXT_PUBLIC` URLs.